UI/volume-control.cpp

```diff
@@ -181,6 +232,9 @@ VolControl::VolControl(OBSSource source_, bool showConfig, bool vertical)
 		controlLayout->addItem(new QSpacerItem(3, 0));
 		// Add Headphone (audio monitoring) widget here
```

Comment:
May be worth removing this line as part of this PR

Reply:
Done